### PR TITLE
Use Bootstrap's collapse for wells.

### PR DESCRIPTION
**Pull request by XhmikosR**

### CODE REVIEW

**Comment by XhmikosR** [comment]
It's not perfect since the toggle button action happens instantly and the collapse has a delay but it's better than before.

Any suggestions/patches welcome.

**Comment by XhmikosR** [comment]
/CC @Johann-S just in case you have time and any improvement ideas.

**Comment by Johann-S** [comment]
Maybe that's related to : https://github.com/twbs/bootstrap/pull/20870 🤔

File: public/assets/js/main.js [hunk 1]
```diff
@@ -8,11 +8,9 @@
         var el = document.querySelectorAll(selector);
 
         function toggleCode(index) {
-            var name = el[index].getAttribute('data-el');
             var btnIcon = el[index].querySelector('span');
 
             el[index].addEventListener('click', function() {
-                document.getElementById(name).classList.toggle('hidden');
                 btnIcon.classList.toggle('caret-open');
```
Comment (Johann-S) [inline public/assets/js/main.js]: Maybe you can do : 

```javascript
setTimeout(function () {
  btnIcon.classList.toggle('caret-open');
}, 350);
```

`350` is the transition duration of the Collapse plugin (https://github.com/twbs/bootstrap/blob/v3-dev/js/collapse.js#L35)

**Comment by XhmikosR** [comment]
I haven't tried that but I will, thanks! I was trying to use transionend
but didn't spend a lot of time.

**Comment by jmervine** [comment]
:+1:

**Pull request merged**